src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -86,6 +121,14 @@ public HubConnection(IConnectionFactory connectionFactory, IHubProtocol protocol
             }
         }
 
+        /// <summary>
+        /// Subscribes a handler that will be raised when a hub method with the specified method and parameters is invoked.
+        /// </summary>
+        /// <param name="methodName">The name of the hub method to subscribe to.</param>
```

Comment:
`The name of the hub method to define`

Technically, the `.On` method is basically like "defining" a method on the client. I wonder if we should use that terminology a bit more.